Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/Definitions/Input.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+
+namespace Frends.Salesforce.CreateSObject
+{
+    /// <summary>
+    /// Input-class for CreateSObject-task.
+    /// </summary>
+    public class Input
+    {
+        /// <summary>
+        /// Salesforce Domain.
+        /// /services/data/v52.0/query will be added automatically, so the domain is enough.
+        /// </summary>
+        [DefaultValue(@"https://example.my.salesforce.com")]
+        [DisplayFormat(DataFormatString = "Text")]
```

Comment:
No need for this line, since the property is of type string. Frends will automatically set the input-type to Text.